### ISSUE TITLE
location issues

### ISSUE THREAD

**Issue by HistoryDiver** [ShammyLevva/FTAnalyzer#144]
Hi, I'm not sure whether this is a bug or simply the way the data is analysed, but reporting here anyway! I'm using the Mac version 1.3.0 (54). 
Previously one of the Admins (Alexander?) explained that locations have to be fixed in the GEDCOM file to enable FTA to display correctly. I don't have desktop software, only an Ancestry subscription, so it was a case of laboriously going through all the records in my tree, fixing the locations as best I could by street (when known), district, town, county, country, separated by commas. That's a big old task, and there are still inevitable differences in addresses - county boundaries change over the years, some locations are more specific than others. it would be great if FTA was able to be a little more sensitive to locations.

I uploaded what I thought was a fixed GEDCOM now, but clicking on the birth location tab still doesn't list locations alphabetically. The most important location is the town or village of birth, But in the results a street address tends to confuse the search bots, and changing cpounty names/boundaries really throw a spanner in the works!
 
Is there no easy way to search for individuals just by Town Name, which tend to be consistent? 

e.g., I've heritage in Rhosllanerchrugog, a town in Denbighshire, North Wales. Locals usually refer to it as "Rhos", but in all the entries in my tree I use the full name. Rhos was previously known in the early 19thC by the anglicized name Moreton Above and Moreton Below. For records that use these I've listed both names, e.g. "Moreton Above, Rhosllanerchrugog". 
Rhos is a spread out town, and has sub-areas such as Ponciau and Johnsontown. This adds an extra level of confusion. Also, although the parish comes under Wrexham, the area used to be with Ruabon, older records especially reflect this. Finally there's the county name - Denbighshire, but at times it's been Clwyd when the Welsh counties were re-organised. 

County boundaries changing are even worse with some other towns, (e.g. some Birmingham suburbs!). FTA doesn't seem to have much sensitivity to pick up on these things at the moment. Can you suggest anything? 

So, it's a minefield, before we even get to street addresses! In my tree the one thing consistent is the town name Rhosllanerchrugog - that appears in every record for the area. Is there no way I can just isolate all the entries with this town name? 

Ultimately a searchbox function is probably the only way to go, is this on the cards?

**Comment by ShammyLevva** [comment]
In the locations report if you select the appropriate level tab then double click an entry it will show you all the individuals and families with that entry. 

So for example on the region tab you should have Denbighshire, Wales double clicking that will show everyone in that location. This makes it easier to find people who have a problematic location and therefore fix it on your ancestry tree. As it is extremely quick to just use search on ancestry to pull up a person for editing. 

If you go to the sub regions tab you should then have towns within the region so Rhos... will be there again a double click shows everyone at that location. 

Places tab would have the Johnstown, Rhos..., Denbighshire, Wales. Etc

So basically use the tabs to filter to the appropriate levels.
